### ISSUE TITLE
Redownload a film/episode when an issue is added

### ISSUE THREAD

**Issue by tomerpeleg-pub** [sct/overseerr#2682]
### Description

The issues feature is great, but I find most people still message me directly since it gets a faster response 🤷. Personally when I get an issue I pretty much always just redownload the affected films/episodes so it would be nice to automate the process. 

### Desired Behavior

Add an option to auto re-search for an episode/movie when an issue is created, similar to the auto-approve feature for normal requests. Maybe add an option for selecting the issue types this should be triggered for (i.e. Video/Audio/Subtitle/Other). 

When an issue is created and matches the selected issue type a request should be sent to sonarr/radarr to mark the download as failed, blacklist the download that was used for the affected episodes/movie and trigger a new search for them.

Not strictly needed, but it would be nice if it also marked the issue as in progress, or left a comment to show it is being actioned, and then closed the issue and/or notified the user when the affected files have been replaced.

I'm not sure if this is in the pipeline, but if Bazaar support is added in the future then potentially support could be added to automatically search for a new subtitle there too if the issue type is set to Subtitle (or even better, let the user select a new one). 

### Code of Conduct

- [X] I agree to follow Overseerr's Code of Conduct

**Comment by Gartonious** [comment]
This would be a nice feature to be honest but it needs to be able to be controlled too, maybe a setting in so people can either choose to have it enabled where it will auto download, enabled so it pings the admins a message asking to redownload or to turn the feature off for those who wants to go out and manually download the file.

**Comment by brandan-schmitz** [comment]
I agree that this would be a helpful feature. I am not so sure about the automatic blacklist/redownload but at least a button somewhere either on an open issue page or on the movie/episode page itself that allows for triggering the process through Sonarr/Radarr.

**Comment by Emalton** [comment]
Is there people already working on this feature? Would like to contribute code if I can.

**Comment by TheCatLady** [comment]
So, Overseerr will never _automatically_ blacklist or redownload media. However, we may offer options in Overseerr to manually perform these tasks.

@Emalton PRs and contributions are always welcome! Feel free to hop into our Discord server to discuss :)

**Comment by Emalton** [comment]
Commenting so this didn't get closed!

**Comment by Emalton** [comment]
@TheCatLady
I think that the blacklisting would be a necessary feature if this were to be implemented as the *arrs handle searching for the media within the parameters

**Comment by Emalton** [comment]
Keep forgetting about this! Would be a great feature.

**Comment by marcodejongh** [comment]
Someone described how to mark as failed using the rest API here: https://www.reddit.com/r/sonarr/comments/nm36np/comment/gzq65do/?utm_source=reddit&utm_medium=web2x&context=3 
can be helpful to whoever decides to implement this issue.
I found it a bit hard to figure out myself using just the rest api documentation.

**Comment by MrN0b0dy98** [comment]
Still interested!🙂

**Comment by Emalton** [comment]
Maybe this would be better instead.

https://github.com/ManiMatter/decluttarr
